sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -1,7 +1,10 @@
 # Copyright lowRISC contributors.
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
-
```

Comment:
I'll revert this before merging, I don't want to trigger CI before getting reviews.